paddle/fluid/framework/init.cc

```diff
@@ -64,7 +64,7 @@ void InitP2P(int count) {
 #endif
 }
 
-void InitDevices() {
+void InitDevices(bool init_p2p) {
```

Comment:
It seems that `InitDevice` is always called with `init_p2p=true`. Do we plan to change some of these invocations to be with false?

Reply:
see __init__.py, For all tests, it should be false

Reply:
https://github.com/PaddlePaddle/Paddle/pull/9699/files#diff-d4bed27d98cf7849623c79d9d2d3c32bR114

Reply:
I see. Cool!